src/main/java/io/wisetime/connector/patrawin/PatrawinDao.java

```diff
@@ -48,8 +67,27 @@ void asTransaction(final Runnable runnable) {
    */
   List<Case> findCasesOrderedByCreationTime(final Instant createdOnOrAfter, final List<String> excludedCaseNumbers,
                                             final int maxResults) {
-    // TODO
-    return ImmutableList.of();
+    final StringBuilder query = new StringBuilder(
+        "SELECT TOP (:maxResults) Arendenr AS CaseNum, Slagord AS Description, Skapatdat AS CreatedDate " +
+        "FROM ARENDE_1 WHERE Skapatdat >= :createdOnOrAfter"
+    );
+    if (!excludedCaseNumbers.isEmpty()) {
+      query.append(" AND Arendenr NOT IN (:excludedCaseNumbers)");
+    }
+    query.append(" ORDER BY Skapatdat, Arendenr");
+
+    SelectQuery selectQuery = query().select(query.toString())
+        .namedParam("maxResults", maxResults)
+        .namedParam("createdOnOrAfter", dbDateTimeUtcFormatter.format(createdOnOrAfter));
+    if (!excludedCaseNumbers.isEmpty()) {
+      selectQuery = selectQuery.namedParam("excludedCaseNumbers", excludedCaseNumbers);
```

Comment:
Its builder pattern - no need to reassign `selectQuery` variable. If you accept this change - `selectQuery` can be marked `final`.
```suggestion
      selectQuery.namedParam("excludedCaseNumbers", excludedCaseNumbers);
```